src/LaraCart.php

```diff
@@ -483,16 +486,20 @@ public function total($format = true, $withDiscount = true)
      *
      * @param boolean $format
      * @param boolean $withDiscount
+     * @param boolean $withTax
      *
      * @return string
      */
-    public function subTotal($format = true, $withDiscount = true)
+    public function subTotal($format = true, $withDiscount = true, $withTax = false)
```

Comment:
Long story short, i previously had it with / without tax. And removed it and move laracart to 1.1

Also just a quick rule : 
https://community.intuit.com/questions/1155144-subtotal-with-tax-and-without-tax